gulpfile.js

```diff
@@ -119,11 +120,12 @@ const getTextStyles = () => {
 gulp.task('template:color', () => {
   const getColors = () =>
     tokensWithType('color').map(color => {
-      const newColor = JSON.parse(JSON.stringify(color));
-      newColor.name = `bpk${pascalCase(
-        newColor.name.replace(newColor.type.toUpperCase(), ''),
+      const colorObject = JSON.parse(JSON.stringify(color));
+      colorObject.name = `bpk${pascalCase(
+        colorObject.name.replace(colorObject.type.toUpperCase(), ''),
       )}`;
-      return newColor;
+      colorObject.value = tinycolor(colorObject.value).toHexString();
```

Comment:
As the the android [https://developer.android.com/reference/android/graphics/Color.html](docs), accepted formats are
 ```
 #RRGGBB
 #AARRGGBB
```
in the token the alpha is at the end of the colour string, given the libray we're using for colours does't allow custom formatting, we'll use `RRGGBB` format

Reply:
So this means we can't support colors with alpha?

Reply:
Not with this version of the tokens, if we ship colours with alpha we will change the script